ROLES.md

```diff
@@ -83,6 +84,17 @@ Here is the set of roles we use within the Istio community, the general responsi
     <td>n/a</td>
     <td>‘approver’ entry in one or more OWNERS files within the istio/contrib repo</td>
   </tr>
+
+  <tr>
+    <td><a href="#sponsor">Sponsor</a></td>
+    <td>Existing member that vouches for another GitHub user that wishes to become a member of the Istio organization</td>
+    <td>
+        <p>Must have close interactions with the prospective member - e.g. code/design/proposal review, coordinating on issues, etc.</p>
+        <p>Must be reviewers or approvers in at least 1 OWNERS file (in any repo in the Istio GitHub organization).</p>
```

Comment:
Any repo ? Won’t it pave the way for someone to be part org by getting two docs owners to sponsor and then the person requests team membership from istio-hackers etc ?

Reply:
I just took that text from the issue template.

@linsun ?

Reply:
I got it from k8s... I think our doc owners bar is not low today, there is only a very few selectively folks, except the Chinese doc site.  I'd be fine if we feel this is an issue and state that we exclude the istio.github.io repo.

Reply:
I agree with excluding istio.github.io from the allowed sponsors.

Reply:
Added an exception for istio.github.io